Documentation/Global/Image.rst

```diff
@@ -9,73 +9,147 @@ Image ViewHelper `<f:image>`
 
 ..  typo3:viewhelper:: image
     :source: ../Global.json
-    :display: tags,description,gitHubLink,arguments
+    :display: tags,description,gitHubLink
+    :noindex:
 
-..  _typo3-fluid-image-example:
+..  contents:: Table of contents
+
+..  _typo3-fluid-image-src:
+
+Argument src - displaying images from the extension assets
+==========================================================
+
+Images used for design purposes and not content purposes are commonly shipped
+with an extension. These images cannot be edited by backend users. They can be
+displayed using the `<f:image>` ViewHelper using the
+:ref:`src <t3viewhelper:viewhelper-argument-typo3-cms-fluid-viewhelpers-imageviewhelper-src>`
+argument and the `EXT:` syntax:
+
+..  code-block:: html
+
+    <f:image src="EXT:my_site_package/Resources/Public/images/typo3_logo.png" alt="alt text" />
+
+The assets **must** be stored in folder :path:`Resources/Public/` or a sub
+folder of an extension.
```

Comment:
"sub folder of an extension"? So `EXT:cars/images` is also allowed? Maybe you mean sub folder of the Resources/Public folder, right?